test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -4544,6 +4549,468 @@ describe('ThirdPartyRegistry', function () {
     })
   })
 
+  describe('reviewThirdPartyWithRoot', function () {
+    it('should update the third party and emit the reviewed event when consume slots is empty', async function () {
+      await thirdPartyRegistryContract.addThirdParties(
```

Comment:
Maybe you can move this to a `beforeEach` clause inside the `describe` if it is being used multiple times like in `reviewThirdParties`